docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -44,13 +44,15 @@ Note how changing the value of the start date section will call `onChange` even
 
 ## Pickers lifecycle
 
-### When is `onClose` called?
+### `onClose`
```

Comment:
WDYT about dropping the code block for these headings where the callback is the only word? 🤔 
```suggestion
### onClose
```